GuildedRoseLLC/Constants.swift

```diff
@@ -0,0 +1,3 @@
+public enum StoryboardIDs {
+    public static let SEGUE__INDEX_TO_DETAIL = "DetailsPageSegue"
```

Comment:
In terms of functionality, this is what I was looking for. A couple small suggestions:

 * Is this the "index to details" segue? Or is it the "details page segue"? Right now, you have two names for the same thing.
 * Conventionally, constants are named as `UpperCamelCase` in Swift. I would make sure to look up and follow naming conventions whenever using a new language.